rootcmd/root.go

```diff
@@ -39,7 +39,7 @@ func New(cnfg *hazelcast.Config) (*cobra.Command, *config.GlobalFlagValues) {
 		Short: "Hazelcast command-line client",
 		Long:  "Hazelcast command-line client connects your command-line to a Hazelcast cluster",
 		Example: "`hzc` - starts an interactive shell 🚀\n" +
-			"`hzc map --name my-map put --key hello --value world` - put entry into map directly\n" +
+			"`hzc map put --key hello --value world --name my-map` - put entry into map directly\n" +
```

Comment:
Why was the name moved after k/v options?

Reply:
Reverted on https://github.com/hazelcast/hazelcast-commandline-client/pull/61/commits/808431d03755b7dda7649c1176a911453b160d31